hwbench/bench/monitoring_structs.py

```diff
@@ -10,86 +14,79 @@ class MonitorMetric:
 
     name: str
     unit: str
-    # We let an option to init with an initial value
-    # The default is set to None and this value shall be used during comparison or repr
     value: float = field(default=sys.float_info.max, compare=False, repr=False)
     values: list[float] = field(default_factory=list, init=False)
     mean: list[float] = field(default_factory=list, init=False)
     min: list[float] = field(default_factory=list, init=False)
     max: list[float] = field(default_factory=list, init=False)
     stdev: list[float] = field(default_factory=list, init=False)
-    samples: list[float] = field(default_factory=list, init=False)
+    samples: list[int] = field(default_factory=list, init=False)
+    full_name: str = field(default="", init=False)
```

Comment:
why adding a new member ?

Reply:
In a dataclass, fields are all supposed to be declared explicitely to allow proper serialization and stuff like that. So this is merely a fix